engine/consensus/compliance/engine.go

```diff
@@ -64,6 +65,19 @@ func NewEngine(
 	prov network.Engine,
 	core *Core) (*Engine, error) {
 
+	blockResponseQueue, err := fifoqueue.NewFifoQueue(
+		fifoqueue.WithCapacity(defaultBlockQueueCapacity),
```

Comment:
This define a new capacity with maybe 100 as default. 
The capacity for block queue is for buffering all blocks, we don't want to buffer as many range responses. 
We should buffer less relatively.
```suggestion
		fifoqueue.WithCapacity(defaultRangeResponseQueueCapacity),
```